themes/10up-theme/.stylelintrc

```diff
@@ -0,0 +1,17 @@
+{
+	"extends": [
+		"@10up/stylelint-config"
+	],
+	"rules": {
+		"at-rule-no-unknown": [
+			true,
+			{
+				"ignoreAtRules": [
+					"screen",
+					"layer",
+					"tailwind"
```

Comment:
@Nikki-Jones Would `@apply` already be considered known, or does that need to be added?

Reply:
@iansvo yes @apply is known/not tailwind specific